content/blog/twitter-data-download-explanation.en.md

```diff
@@ -0,0 +1,903 @@
+
+If you go to [Twitter's settings page](https://twitter.com/settings/your_twitter_data) for user data without any user being logged in, you'll see a page that looks a bit like this.
+
+<img class="offset-image offset-image-left" src="/img/blog/twitter-data-explanation/non-login-twitter-data-1.png" style="height: 500px; margin-top: 50px; margin-bottom: 100px;"> <img class="offset-image offset-image-right" src="/img/blog/twitter-data-explanation/non-login-twitter-data-2.png" style="height: 500px; margin-top: 50px; margin-bottom: 100px;"> 
+
+
+<a id="twitter-logged-in-procedure">     
+## The Procedure to get the Data Download from Twitter(Logged in user)
+</a>
+
+1. Go to your Account settings by clicking on the more icon in the navigation bar, and selecting Settings and privacy from the menu.
+   Or go to [Twitter's settings page](https://twitter.com/settings/your_twitter_data) for user data after logging in.
+
+2. Under the Account section, click Your Twitter data.
+
+3. Enter your password under Download your Twitter data, then click Confirm.
+   {{< img src="/img/blog/twitter-data-explanation/login-twitter-1.png" caption="Enter password to proceed. Ensure that your email address is verified earlier." >}}  
+4. Click the Request data button. If your Twitter account is connected to Periscope, the option to Request data from Periscope will show up as well.
+   {{< img src="/img/blog/twitter-data-explanation/login-twitter-2.png" caption="Click Request archive on whichever option you want." >}}  
+5. When your download is ready, Twitter will send an email to your email account. When you recieve it, from the link or from [settings](https://twitter.com/settings/your_twitter_data), you can go to the same earlier options and find the ready archive.
+   {{< img src="/img/blog/twitter-data-explanation/login-twitter-3.png" caption="Click on Download Archive." >}}  
+6. Click the Download data.
+   {{< img src="/img/blog/twitter-data-explanation/login-twitter-4.png" caption="Click the Request archive button depending on whichever option you want." >}}  
+   If you do this process from an iOS or Android device, the options are found in the top menu, where you will either see a navigation menu icon/profile icon. From there go to Settings and privacy, then Account, then under Data and permissions, select Your Twitter data and continue from Step 3 onwards.  
+
+
+
+
+You should now have a .zip file downloaded. Extract and Save it. This file contains all your Twitter user data.
+
+
+
+<a id="twitter-data-meaning"></a>
+## What does this data mean?
+
+<img class="offset-image offset-image-right" src="/img/humaaans/question-1.svg" style="height: 350px;">
+<style>
+table, td, th {
+    border-collapse: collapse;
+    border-spacing: 0;
+    border: 2px solid #000000;
+    text-align: center;
+}
+</style>
+<table>
```

Comment:
It would be great if you could give some examples of things in the download that may be particularly interesting to the user before listing the whole table.